src/src/SceneNode.cpp

```diff
@@ -196,4 +216,36 @@ namespace mineola {
         return node.Name() == name_str;
       });
   }
+
+void SceneNode::ComputeAABB(glm::vec3 &center, glm::vec3 &extent) {
+  glm::vec3 min(std::numeric_limits<float>::max()), max(std::numeric_limits<float>::lowest());
+
+  DFTraverse([&min, &max](SceneNode &node) {
+    bool valid_aabb = false;
+    glm::vec3 min_node(std::numeric_limits<float>::max()), max_node(std::numeric_limits<float>::lowest());
+    for (auto renderable : node.Renderables()) {
+      // check min and max vertex is valid or not
+      if (renderable->MinVertex() != glm::vec3() || renderable->MaxVertex() != glm::vec3()) {
```

Comment:
check is renderable has valid AABB (i.e. unique_ptr is not null). glm::vec3() is a valid vertex value, shouldn't be used as a flag.